mande/chain.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mande",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://mande.network",
+  "pretty_name": "Mande Network",
+  "chain_id": "mande_18071918-1",
+  "bech32_prefix": "mande",
```

Comment:
Please register this prefix:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Raised a PR - https://github.com/satoshilabs/slips/pull/1753

Reply:
This is merged @JeremyParish69